tools/build/libwebrtc/build.ps1

```diff
@@ -157,13 +159,10 @@ function Write-GnArgs([string]$Platform, [string]$Arch, [string]$Config, [string
     switch -Exact ($Platform) {
         "Win32" {
             $target_os="win"
-            $extra="_win32"
         }
         "UWP" {
             $target_os="winuwp"
-            $extra="_uwp"
-            $UwpContent = @"
-
+            $extra_content=@"
 # Use WinRT video capture for UWP
 rtc_win_video_capture_winrt=true
```

Comment:
I just remembered this: the latest recommendation is using the WinRT capture module on win32 too.

Reply:
Fixed.